### DISCUSSION TITLE
mongoclient 자체 sort 기능 부활?

### DISCUSSION THREAD

**Discussion by utawakuworker** [yiminwook/liveuta-asis#72] [Q&A]
ScheduledTime 파싱하는거 해결 됐으니까 이제 자체 sort() 기능 부활시켜도 되지 않아? 그 소트가 뭔가 제대로 안돼서 커스텀으로 만든거였나?

추가로 채널조회에서 보이는것도 정렬해달라는 요청이 있어서, 이쪽도 search에서 받아올때 sort 하면 좋을듯....

**Comment by yiminwook** [comment]
지금 몽고db에서 이미 시간으로 sort중임 서버리스 함수에서도 채널 이름 두가지로 sort중인데 몽고db에서 sort 한번에 여러개 되는거 확인되면 뺄예정임, 이거 확인되면 전체적으로 적용시킬게

**Comment by yiminwook** [comment]
mongoDB에서 채널이랑, 스케쥴 가져올때 sort하는거로 수정했음 스케쥴시간, 한국채널명 순으로 정렬됨
혹시 안되는 부분있으면 알려줘
